_security/multi-tenancy/dynamic-config.md

```diff
@@ -0,0 +1,106 @@
+---
+layout: default
+title: Dynamic configuration in OpenSearch Dashboards
+parent: OpenSearch Dashboards multi-tenancy
+nav_order: 147
+---
+
+
+# Dynamic configuration in OpenSearch Dashboards
+
+Dynamic configuration of multi-tenancy in OpenSearch Dashboards provides options to make common settings for tenancy without having to make changes to the configuration YAML files on each node and then restart the cluster. You can take advantage of this functionality by using the Dashboards interface or the REST API. The following list includes description of the options currently covered by dynamic configuration:
+
+- **Disable or enable multi-tenancy** - Administrators can disable and enable multi-tenancy dynamically. Disabling multi-tenancy does not pose a risk for loss of data. If and when an administrator chooses to re-enable tenancy, all previously saved objects are preserved and made available.  The default is `true`.
+  
+  This setting does not have an impact on the global tenant, which always remains enabled.
+  {: .note }
+
+- **Disable or enable private tenant** - This option allows administrators to disable and enable private tenants. As with the enable multi-tenancy setting, when private tenants are re-enabled all previously saved objects are preserved and made available.
+- **Default tenant** - This option allows an administrator to choose either a global, private, or custom tenant as the default when users sign on. In the case where a user doesn't have access to the default tenant (for example, if a custom tenant unavailable to the user was specified as the default), the default transitions to the preferred tenant, which is specified by the `opensearch_security.multitenancy.tenants.preferred` setting in the `opensearch-dashboards.yml` file. See [Multi-tenancy configuration]({{site.url}}{{site.baseurl}}/security/multi-tenancy/multi-tenancy-config/) for details about this setting.
+
+Once you make changes to multi-tenancy with dynamic configuration and save, users are logged out of their sessions and must log back in.
```

Comment:
This case is conditional. For eg: 
1. If admin disables multi-tenancy, users are not logged out if they are in Global tenant.
2. If admin disables private tenant, users are logged out only if they were in Private tenant.
3. Users are not logged out if only default tenant is changed.

Reply:
Thanks for the clarification. I've reworded this.